docs/.vitepress/config.js

```diff
@@ -53,6 +53,10 @@ const config = {
           {
             text: 'Passing Data to Components',
             link: '/guide/essentials/passing-data'
+          },
+          {
+            text: 'Write components that are easy to test',
+            link: '/guide/essentials/easy-to-test'
```

Comment:
Not 100% sure this is the right place for this kind of content

Reply:
🤔 

Not sure. Either way, should we merge this one up so it's at least available for people to use?

Reply:
sure! 🚀